<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveSelectedFieldAttributeToCSVDialogBase</class>
 <widget class="QDialog" name="SaveSelectedFieldAttributeToCSVDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Selected Field Attribute to CSV</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Layer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLayers"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lstFields"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSelectFields">
       <property name="text">
        <string>Select Fields</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveFields">
       <property name="text">
        <string>Remove Fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtOutputFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseOutput">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSaveCSV">
     <property name="text">
      <string>Save CSV</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>